src/styles/index.scss

```diff
@@ -1,3 +1,8 @@
-body {
-  margin: 0;
-}
+@import './vars';
+@import './page';
+@import './stars';
+@import './card';
+@import 'imac_photo';
+@import './rewiews';
```

Comment:
The import statement './rewiews' seems to contain a typo. It should likely be './reviews' to match the corrected class name in the HTML.